libbpf-tools/sigsnoop.bpf.c

```diff
@@ -72,7 +84,6 @@ int kill_entry(struct syscall_trace_enter *ctx)
 {
 	pid_t tpid = (pid_t)ctx->args[0];
 	int sig = (int)ctx->args[1];
-
```

Comment:
Please keep this empty line here. We would like an empty line between declaration and other codes.

Reply:
Fixed.